### DISCUSSION TITLE
Formulários multi-coluna

### DISCUSSION THREAD

**Discussion by wwwflorencio** [frontendbr/forum#247] [Dicas]
Estava vendo um [post bem legal sobre UX e formulários](https://uxdesign.cc/design-better-forms-96fadca0f49c#.h6h4j2ng9), e acabei me deparando com um item que dizia a respeito sobre formulários multi-coluna, que me impressionou um pouco pelo fato de eu nunca ter visto nada falando a respeito.

Queria saber a opinião de vocês a respeito (:

**Comment by iamgbayer** [comment]
Realmente, tem umas dicas maneiras e até explica em onde vai afetar, algumas eu acabei concordando, agora teve outras que não reparei muito no empacto na experiencia, acredito que seja mais relacionado a estética.

**Comment by htrgouvea** [comment]
Concordo com você @guuibayer!

**Comment by ortizvinicius** [comment]
Acho que no caso usar uma coluna é melhor pois alguns usuários usam o mouse pra selecionar o campo, ou então o dedo mesmo, aí a pessoa ja vai no embalo sabe, quando se usa mais colunas você perde esse movimento natural, da a impressão de que uma coluna "cansa" menos.

**Comment by ortizvinicius** [comment]
Claro que há casos e casos, poucas regras são absolutas, tudo deve ser sempre analisado.
Me lembra de quando o tableless "começou" aí o pessoal ia fazer uma tabela e usava div, pq tinha que ser tableless kk

**Comment by felipefialho** [comment]
Li ontem e por curiosidade estava conversando sobre isso com o Designer da empresa.

Concordo com o artigo. Acho que formulários multi colunas só são válidos em campos que são dependentes. Já vamos começar a adequar nossos layouts para esse pensamento.

**Comment by lagden** [comment]
Achei um pouco agressivo...

O primeiro já não iria dar muito certo!!
vamos trocar os nomes dos animais para: `dromedário, capivara, tamandua, cachorro, lobo-guará`... já cagou tudo!!!

Até acho bonito desse forma se for no máximo 3 opções.

---

Tem até um conflito de idéias!!!
Ele diz: `Forms should be one column` e depois me manda `Use field length as an affordance`
Bahh!! esse form vai ficar bonito!!! hehehe...

---

Pessoas não gostam de formulários!! O importante é tentar `"gameficar"` ou deixar bem agradável de se preencher.

Exemplo de um projeto que faço: https://demo.nimble.com.br/NEO?theme=trello
É um formulário gigante, mas você nem percebe!!!

**Comment by felipefialho** [comment]
Porra, bem bacana @lagden
